tests/test_pytweet.py

```diff
@@ -1,5 +1,55 @@
 from pytweet import __version__
-from pytweet import pytweet
+#from pytweet import pytweet
+from pytweet.pytweet import get_tweets
+from pytweet.pytweet import plot_timeline
+import pandas as pd
+import pytest
 
 def test_version():
-    assert __version__ == '0.1.0'
\ No newline at end of file
+    assert __version__ == '0.1.0'
+
+def helper_create_data():
+    """
+    Helper function for creating dataframe for testing
+    Parameters
+    -----------
+    Returns
+    --------
+    pandas.DataFrame
+        Returns a dataframe to be used for testing
+    Examples
+    ---------
+    >>> helper_create_data()
+    """
+    tweet_data = pd.read_csv("./tests/trumptweets-test.csv")
+    return tweet_data
+
+def test_plot_timeline():
+    """
+    Tests the plot_timeline function to make sure the outputs are correct.
+    Returns
+    --------
+    None
+        The test should pass and no asserts should be displayed.
+    """
+    # Calling helper function to create data
+    data = helper_create_data()
+    
+    # Test the Exception is correctly raised when the type of `df`
+    # argument is wrong.
+    with pytest.raises(Exception) as e:
+        assert plot_timeline('', 'date')
```

Comment:
```suggestion
        plot_timeline('', 'date')
```

assert doesn't really do anything here. Also might wanna consider indent the next line to fall under `with`

Reply:
Thanks, changed!

Reply:
Changed!